docs/pages/multiple-renderer.page.server.mdx

```diff
@@ -1,4 +1,9 @@
-import { Link } from '@brillout/docpress'
+import { Link, Warning } from '@brillout/docpress'
+
+<Warning>
+  Having multiple `/renderer` folders is deprecated in favor of having `+*.js` files in higher page folders,
+  see <Link href="/default-page" />
+</Warning>
```

Comment:
@brillout is this fine with you? I have reworked `/default-page` to teach that

Reply:
👍 I like the idea of showcasing default settings increasingly as `/pages/+someConfig.js` and less and lass as `/renderer/+someConfig.js`. It makes sense since less and less users are going to have a `renderer/` directory as they'll increasingly use `vike-{react,vue,solid}`.

That said, for users with custom renderers, such as Ecosia, I'm thinking `renderer/` can still make sense. IMO tucking the rendering logic into a `renderer/` directory is a neat way to seperate render logic from defining pages.

Also I'm increasingly thinking that it would be neat to support `$ eject vike-react` which would move the renderering logic from `node_modules/vike-react/src/renderer` to `renderer/` so that the user can gain full control. So I'm thinking the `renderer/` directory could also be useful in that context.

So while I agree that we should showcase `renderer/` less and less, I think it would be nice to still use it in docs that showcase custom renderers. Or, alternatively, we can just mention it as a note "Btw, for a clear separation of concerns, you can create a `renderer/` directory in order to...") .

WDYT?

Reply:
I think there is a slight misunderstanding. This page isn't about the root `/renderer` folder but about some `/**/renderer` subfolders, which I think aren't so useful, as things can just be inside `/**/pages` instead. I'm still happy with advertizing a root `/renderer` folder. What do you think?

Reply:
I see, although I think the following POV still holds:

> That said, for users with custom renderers, such as Ecosia, I'm thinking `renderer/` can still make sense. IMO tucking the rendering logic into a `renderer/` directory is a neat way to seperate render logic from defining pages.

E.g. the classic example of having an admin panel rendered as SPA while having landing/marketing pages rendered with SSR.

I think it's neat to have these two very different rendering logics live in two separate `renderer/` directories.

Reply:
alright, makes sense, I'll revert the deprecation notice and adapt that page too

Reply:
fixed